bridge/client/app/_views/ktb-dashboard-view/ktb-dashboard-view.component.spec.ts

```diff
@@ -1,18 +1,18 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { DashboardLegacyComponent } from './dashboard-legacy.component';
-import { DataService } from '../_services/data.service';
+import { KtbDashboardViewComponent } from './ktb-dashboard-view.component';
+import { DataService } from '../../_services/data.service';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { ApiService } from '../_services/api.service';
-import { ApiServiceMock } from '../_services/api.service.mock';
+import { ApiService } from '../../_services/api.service';
+import { ApiServiceMock } from '../../_services/api.service.mock';
 import { skip, take } from 'rxjs/operators';
-import { ProjectSequences } from '../_components/ktb-project-list/ktb-project-list.component';
-import { POLLING_INTERVAL_MILLIS } from '../_utils/app.utils';
+import { ProjectSequences } from './ktb-project-list/ktb-project-list.component';
+import { POLLING_INTERVAL_MILLIS } from '../../_utils/app.utils';
 import { RouterTestingModule } from '@angular/router/testing';
 import { Navigation, Router } from '@angular/router';
 
-describe('DashboardLegacyComponent', () => {
-  let component: DashboardLegacyComponent;
-  let fixture: ComponentFixture<DashboardLegacyComponent>;
+describe('DashboardLegacyView', () => {
```

Comment:
Could you also rename it to `DashboardView`, `KtbDashboardView`or `KtbDashboardViewComponent`?